test/functional/page_objects/dashboard_page.ts

```diff
@@ -606,11 +611,6 @@ export class DashboardPageObject extends FtrService {
     return panels.length;
   }
 
-  public async getAllPanels() {
-    this.log.debug('getAllPanels');
-    return await this.testSubjects.findAll('embeddablePanel');
-  }
-
```

Comment:
This function was accidentally duplicated in a previous PR - should use the pre-existing `getDashboardPanels` instead.